app/components/dropdown-widget/component.js

```diff
@@ -172,47 +177,64 @@ export default Ember.Component.extend({
             }
 
             let filter_data = await Ember.$.ajax({
-                url: 'https://dev-labs.cos.io/api/records/search/creativeworks/_search?request_cache=true',
+                url: 'https://dev-labs.cos.io/records/_search?request_cache=true',
                 crossDomain: true,
                 data: JSON.stringify(filter_query),
                 type: 'POST',
                 contentType: 'application/json'
             });
-            let afilteredList = filter_data.hits.hits.map(function(x) {
-                if (widget_category === "contributors") {
+
+            this.send('filterInput' ,widget_category, filter_data , search_term)
```

Comment:
this schedules the next call to occur in the runloop. In this case, we already have an run loop that we're in, so it may be less overhead to just call the fn directly from here.

Reply:
fixed